ui-tests/src/test/java/org/openmrs/reference/page/MarkPatientDeceasedPage.java

```diff
@@ -0,0 +1,124 @@
+package org.openmrs.reference.page;
+
+import static org.junit.Assert.assertTrue;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+
+public class MarkPatientDeceasedPage extends Page {
+
+    private static final String URL = "/coreapps/markPatientDead.page";
+    private static final By DICTIONARY_LINK= By.id("dictionaryNavLink");
+    private static final By ADD_NEW_CONCEPT = By.linkText("Add new Concept");
```

Comment:
is ADD_NEW_CONCEPT a button or a link or a filed ? Also retrieving an element text is via linkText() is not a good deal because they can easily break, opt for id() or cssSelcetor()

Reply:
What about via xPath() ?

Reply:
xpath breaks the test at some point